x-pack/platform/plugins/shared/streams/server/routes/streams/significant_events/route.ts

```diff
@@ -204,7 +204,7 @@ const generateSignificantEventsRoute = createServerRoute({
       generateSignificantEventDefinitions(
         {
           definition,
-          system: params.body.system,
+          system: params.body?.system,
```

Comment:
the type doesn't account for this?

Reply:
Nope, for some reason `body` isn't marked as possibly undefined, and both Kibana router and Zod are happy if it's missing.

So I got an error here because I didn't pass any `body` param when using the client, and the type didn't warn me, so somewhere the types are wrong.